F1WM.UnitTests/Controllers/NewsControllerTests.cs

```diff
@@ -79,5 +79,92 @@ public async Task ShouldReturnEmptyListOfImportantNews()
 			serviceMock.Verify(s => s.GetImportantNews(), Times.Once);
 			Assert.Empty(result);
 		}
+
+		[Fact]
+		public async Task ShouldReturnNewsTypes()
+		{
+			await controller.GetTypes();
+
+			serviceMock.Verify(s => s.GetNewsTypes(), Times.Once);
+		}
+
+		[Fact]
+		public async Task ShouldReturnNewsTags()
+		{
+			await controller.GetTags();
+
+			serviceMock.Verify(s => s.GetNewsTags(), Times.Once);
+		}
+
+		[Fact]
+		public async Task ShouldReturnNewsCategories()
+		{
+			await controller.GetCategories();
+
+			serviceMock.Verify(s => s.GetNewsCategories(), Times.Once);
+		}
+
+		[Fact]
+		public async Task ShouldReturnNewsTagsByCategoryId()
+		{
+			var categoryId = 40;
+
+			await controller.GetTags(categoryId);
+
+			serviceMock.Verify(s => s.GetNewsTagsByCategoryId(categoryId), Times.Once);
+		}
+
+		[Fact]
+		public async Task ShouldReturnNewsByTypeId()
```

Comment:
I think it's worth to also test cases when page and countPerPage are explicitly specified.